### ISSUE TITLE
Handling infe items of type 'uri '

### ISSUE THREAD

**Issue by dukesook** [strukturag/libheif#928]
The `Box_infe` class already contains the `std::string m_item_uri_type` variable, but no getters/setters for it. The
`HeifContext::add_generic_metadata()` function could be easily extended to handle this capability. Would there be interest if I created a pull request for this functionality?

**Comment by farindk** [comment]
Implemented in 846f1a84ce87abed724c0e095d43d7ef9357732a and 71cb4396814b1236e5bf09faf17f98b517c410c2.
Please check if this meets your requirements.

**Comment by bradh** [comment]
Looks fine for me. @dukesook anything more?

**Comment by dukesook** [comment]
@farindk Yes, the heif_item functions look like a great. Is it still possible to merge these into master?
